### ISSUE TITLE
small fiber neuropathy

### ISSUE THREAD

**Issue by Genetics-CK** [obophenotype/human-phenotype-ontology#11602]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Abnormal peripheral nervous system morphology HP:0000759

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:658549

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@Genetics-CK Can you provide more context? "Neuropathy" is a general term to describe a disease state, not an actual clinical observation which would be in scope for HPO.
